dbt_copilot_helper/templates/pipelines/environments/manifest.yml

```diff
@@ -17,7 +17,7 @@ source:
   properties:
     # Todo: Allow for overriding this, but without risking deploying a branch to higher environments
     branch: main
-    repository: {{ git_repo }}
+    repository: https://github.com/{{ git_repo }}
```

Comment:
I wonder if we should add the `uktrade` organisation as part of this, so it's a bit more specific.

Reply:
Maybe, though I'm hesitant to hard code to that.